app/core/models.py

```diff
@@ -15,9 +15,19 @@
 
 class User(AbstractBaseUser, PermissionsMixin):
     """User in the system."""
+
```

Comment:
I think we should create a Nested class named `Role` here, with constants for values:
```
ADMIN = 'PortalAdmin'
RECRUITER = 'Recruiter'
APPLICANT = 'Applicant'
# with the following choices
CHOICES = (
        ('Admin', 'PortalAdmin'),
        ('Recruiter', 'Recruiter'),
        ('Applicant', 'Applicant'),
)
```

Reply:
updated 👍